### PR TITLE
Implement compile time and runtime `bin`

### PR DESCRIPTION
**Pull request by namannimmo10**

### CODE REVIEW

File: integration_tests/run_tests.py
Comment (namannimmo10) [inline integration_tests/run_tests.py:30]: I will move this to "LPython + CPython tests" once we fix #175

**Review by certik — APPROVED** [review]
I think this looks good, it adds a compile time as well as a runtime implementation.

Would you please mind waiting until #181 is merged, and then resolving conflicts? I need that one to be merged, as I made a mistake and forgot to test it.

**Pull request merged**
